projects/novo-elements/src/utils/form-utils/FormUtils.ts

```diff
@@ -835,24 +835,23 @@ export class FormUtils {
     }
   }
 
-  /**
-   * Get the min start date of a Date base on field data.
-   */
-  private getStartDate(field: any): Date | null {
-    if (field.allowedDateRange) {
-      return this.getStartDateFromRange(field.allowedDateRange);
+  private getEndDateFromRange(dateRange: { maxDate: string; minOffset: number }): Date {
+    if (dateRange.maxDate) {
+      return dateFns.parse(dateRange.maxDate);
+    } else if (dateRange.minOffset) {
+      return dateFns.addDays(dateFns.startOfToday(), dateRange.minOffset);
     }
-    // there is no restriction on the start date
-    return null;
   }
 
-  private inferStartDate(controlConfig, field) {
-    if (field.dataType === 'Date') {
-      const startDate = this.getStartDate(field);
-      if (startDate) {
-        controlConfig.startDate = startDate;
-      }
-      return startDate;
+  /**
+   * Get the min start date and max end date of a Date base on field data.
+   */
+
+  private inferDateRange(controlConfig, field): void {
+    if (field.dataType === 'Date' && field.allowedDateRange) {
+      controlConfig.startDate = this.getStartDateFromRange(field.allowedDateRange);
+      controlConfig.endDate = this.getEndDateFromRange(field.allowedDateRange);
+      controlConfig.customTooltip = field.allowedDateRange?.customTooltip;
```

Comment:
See above - this should just be tooltip, not customTooltip.

Reply:
changed it to be disabledTooltip. I was using tooltip, however in ControlTemplate.ts we assign this tooltip to other form elements, so this value was popping up at un-related places.. therefore, I had to create an other key for this sake.